tslint.js

```diff
@@ -1,64 +1,64 @@
-exports.configuration = {
-    "rules": {
-        "class-name": true,
-        "curly": true,
-        "eofline": true,
-        "indent": [
-            true,
-            "spaces"
-        ],
-        "max-line-length": [
-            false,
-            140
-        ],
-        "member-ordering": [
-            true,
-            "variables-before-functions"
-        ],
-        "no-conditional-assignment": true,
-        "no-consecutive-blank-lines": true,
-        "no-console": [
-            true,
-            "debug",
-            "info",
-            "time",
-            "timeEnd",
-            "trace",
-            "log"
-        ],
-        "no-debugger": true,
-        "no-duplicate-key": true,
-        "no-duplicate-variable": true,
-        "no-empty": true,
-        "no-eval": true,
-        "no-shadowed-variable": false,
-        "no-switch-case-fall-through": true,
-        "no-trailing-comma": true,
-        "no-trailing-whitespace": true,
-        "no-unreachable": true,
-        "no-unused-expression": false,
-        "no-unused-variable": false,
-        "no-use-before-declare": true,
-        "one-line": [
-            true,
-            "check-catch",
-            "check-else",
-            "check-open-brace",
-            "check-whitespace"
-        ],
-        "quotemark": [
-            true,
-            "single",
-            "avoid-escape"
-        ],
-        "semicolon": false,
-        "whitespace": [
-            true,
-            "check-branch",
-            "check-decl",
-            "check-operator",
-            "check-separator",
-            "check-type"
-        ]
-    }
-};
+exports.configuration = {

```

Comment:
Is this file really used? I'm asking because `tsjs` is a dependency

Reply:
I removed the file, tslint wasn't even in the list of dependencies